modules/sitemaps/sitemaps.php

```diff
@@ -1,782 +1,2370 @@
-		'xmlns'              => 'http://www.sitemaps.org/schemas/sitemap/0.9',
-		// Mobile namespace from http://support.google.com/webmasters/bin/answer.py?hl=en&answer=34648
-		'xmlns:mobile'       => 'http://www.google.com/schemas/sitemap-mobile/1.0',
-		'xmlns:image'        => 'http://www.google.com/schemas/sitemap-image/1.1',
-	) );
-}
+	private function sitemap_to_index_item ( $post ) {
+		$url = site_url() . '/' . $post->post_title . '.xml';
+
+		/*
+		 * Must use W3C Datetime format per the spec.
+		 * https://www.w3.org/TR/NOTE-datetime
+		 * Also recall that we stored the most recent modification time
+		 * among all the posts in this sitemap in post_date.
+		 */
+		$last_modified = str_replace( ' ', 'T', $post->post_date) . 'Z';
+
+		$xml =
+			"<sitemap>\n" .
+			" <loc>$url</loc>\n" .
+			" <lastmod>$last_modified</lastmod>\n" .
+			"</sitemap>\n";
+
+		return array(
+			'xml'           => $xml,
+			'last_modified' => $post->post_date
+		);
+	}
 
-/**
- * Start sitemap XML document, writing its heading and <urlset> tag with namespaces.
- *
- * @module sitemaps
- *
- * @param $charset string Charset for current XML document.
- *
- * @return string
- */
-function jetpack_sitemap_initstr( $charset ) {
-	global $wp_rewrite;
-	// URL to XSLT
-	if ( $wp_rewrite->using_index_permalinks() ) {
-		$xsl = home_url( '/index.php/sitemap.xsl' );
-	} else if ( $wp_rewrite->using_permalinks() ) {
-		$xsl = home_url( '/sitemap.xsl' );
-	} else {
-		$xsl = home_url( '/?jetpack-sitemap-xsl=true' );
-	}
-
-	$initstr = '<?xml version="1.0" encoding="' . $charset . '"?>' . "\n";
-	$initstr .= '<?xml-stylesheet type="text/xsl" href="' . esc_url( $xsl ) . '"?>' . "\n";
-	$initstr .= '<!-- generator="jetpack-' . JETPACK__VERSION . '" -->' . "\n";
-	$initstr .= '<urlset';
-	foreach ( jetpack_sitemap_namespaces() as $attribute => $value ) {
-		$initstr .= ' ' . esc_html( $attribute ) . '="' . esc_attr( $value ) . '"';
-	}
-	$initstr .= ' />';
-
-	return $initstr;
-}
 
-/**
- * Load XSLT for sitemap.
- *
- * @module sitemaps
- *
- * @param string $type XSLT to load.
- */
-function jetpack_load_xsl( $type = '' ) {
 
-	$transient_xsl = empty( $type ) ? 'jetpack_sitemap_xsl' : "jetpack_{$type}_sitemap_xsl";
+	/**
+	 * Construct the image sitemap index url entry for an image sitemap post.
+	 *
+	 * @link http://www.sitemaps.org/protocol.html#sitemapIndex_sitemap
+	 *
+	 * @access private
+	 * @since 4.5.0
+	 *
+	 * @param WP_Post $post The image sitemap post to be processed.
+	 *
+	 * @return string An XML fragment representing the post URL.
+	 */
+	private function image_sitemap_to_index_item ( $post ) {
+		$url = site_url() . '/' . $post->post_title . '.xml';
+
+		/*
+		 * Must use W3C Datetime format per the spec.
+		 * https://www.w3.org/TR/NOTE-datetime
+		 * Also recall that we stored the most recent modification time
+		 * among all the posts in this sitemap in post_date.
+		 */
+		$last_modified = str_replace( ' ', 'T', $post->post_date) . 'Z';
```

Comment:
Do you mean `$last_modified`? It is used in line 1194 when we interpolate `$xml`.